base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -4,6 +4,9 @@
 		<entry>
 			<key>Analyser</key>
 			<value>
+				<categories>
+					<category>Monitors</category>
+				</categories>
```

Comment:
Ask other people (esp. controls section) to check the categories in this file are sensible.

Reply:
Agree, I don't know all the devices well enough to check this.
One thing I did notice is that the Chell CCD100 is listed under other - looking at the device specs it seems to be a fairly generic display, however I have only ever heard it being referred to as a pressure transducer in the context of ISIS - should it be in the "Pressure control" category?